atomic_reactor/plugins/pre_check_user_settings.py

```diff
@@ -81,6 +82,14 @@ def operator_bundle_from_scratch(self):
         ):
             raise ValueError(msg)
 
+    def multistage_docker_api_check(self):
+        """Check if multistage build isn't tried with docker_api"""
+        if self.workflow.builder.tasker.build_method != CONTAINER_DOCKERPY_BUILD_METHOD:
+            return
+
+        if len(self.workflow.builder.dockerfile_images.original_parents) > 1:
+            raise RuntimeError("Multistage builds can't be build with docker_api")
```

Comment:
Can we suggest what to use instead? Link to upstream doc, or mention `build_type` in container.yaml must be imagebuilder?

Reply:
added